Header header
float64 velocity
float64 angular_velocity
float64 steering_wheel_angle_can
float64 tire_angle